DQM/BeamMonitor/plugins/AlcaBeamMonitor.h

```diff
@@ -24,25 +24,29 @@
 class BeamFitter;
 class PVFitter;
 
-class AlcaBeamMonitor : public DQMOneLumiEDAnalyzer<> {
+struct NoCache {};
```

Comment:
Given this is a rather common name, please enclose it in a namespace such as
```
namespace alcabeammonitor { struct NoCache {};}
```

Reply:
sure, implementing it now